tests/async_tests/test_interfaces.py

```diff
@@ -204,6 +205,46 @@ async def test_http_proxy(
         assert reason == b"OK"
 
 
+@pytest.mark.parametrize("family", [socket.AF_INET])
```

Comment:
Probably worth just setting it below?

Reply:
As above, this originally tested AF_INET6 also, but the test machine doesn't support IPv6.